web/ASC.Web.Components/src/components/text-input/text-input.test.js

```diff
@@ -33,4 +33,28 @@ describe('<TextInput />', () => {
 
     expect(shouldUpdate).toBe(true);
   });
+
+  it('accepts id', () => {
+    const wrapper = mount(
+      <TextInput value="text" onChange={event => alert(event.target.value)} id="testId" />
+    );
+
+    expect(wrapper.prop('id')).toEqual('testId');
+  });
+
+  it('accepts className', () => {
+    const wrapper = mount(
+      <TextInput value="text" onChange={event => alert(event.target.value)}className="test" />
```

Comment:
Need space between onChange and className

Reply:
Fixed in [commit](https://github.com/ONLYOFFICE/AppServer/pull/12/commits/633e85419febf1b74a158806464a6ad2b0f37052)